Store/Store/WebViewController/WebViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  WebViewController.swift
+//  Store
+//
+//  Created by Анна Сычева on 10.10.2022.
+//
+
+import UIKit
+import WebKit
+
+/// Экран вебвью
+final class WebViewController: UIViewController {
+    
+    // MARK: - Private visual Components
+    
+    private lazy var webView = WKWebView()
+    private lazy var toolBar = UIToolbar()
+    private lazy var backButtonItem = makeButtonItem(systemName: "chevron.left")
```

Comment:
```suggestion
    private lazy var backButtonItem = makeButtonItem(systemName: "")
```
Все текстовки в Константах